### PR TITLE
Dagger

### PR DESCRIPTION
**Pull request by yonbergman**
Wohooo

### CODE REVIEW

File: app/src/main/java/com/iic/mokojin/activities/AddPlayerActivity.java [hunk 1]
```diff
@@ -206,11 +204,11 @@ protected FilterResults performFiltering(CharSequence constraint) {
                         List<Person> filteredPeople = new ArrayList<>();
 
                         if (TextUtils.isEmpty(constraint)) {
-                            filteredPeople = mPeople;
+                            filteredPeople = mPeopleListStore.getPeopleList();
                         } else {
                             constraint = constraint.toString().toLowerCase();
-                            for (int i = 0; i < mPeople.size(); i++) {
-                                Person person = mPeople.get(i);
+                            for (int i = 0; i < mPeopleListStore.getPeopleList().size(); i++) {
```
Comment (giladgo) [inline app/src/main/java/com/iic/mokojin/activities/AddPlayerActivity.java:219]: Why did you remove the caching?

**Comment by udiWertheimer** [comment]
ole

File: app/src/main/java/com/iic/mokojin/activities/AddPlayerActivity.java (hunk 1 above)
Comment (yonbergman) [inline app/src/main/java/com/iic/mokojin/activities/AddPlayerActivity.java:219]: didn't see the need for two levels of caching (the store + other thing)

File: app/src/main/java/com/iic/mokojin/modules/DataModule.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+/**
+ * Created by yon on 3/12/15.
+ */
+@Module(
+    injects = {
+            CurrentMatchFragment.class,
+            PlayerQueueFragment.class,
+            AddPlayerActivity.AddPlayerFragment.class,
+            ChooseCharactersActivity.ChooseCharactersFragment.class,
```
Comment (giladgo) [inline app/src/main/java/com/iic/mokojin/modules/DataModule.java:26]: Need to add `MokojinBroadcastReceiver.class`

**Comment by giladgo** [comment]
ok
